### PR TITLE
[Edit child] The 'ЗБЕРЕГТИ' button stays enabled when mandatory 'По-батькові' field is empty #962

### PR DESCRIPTION
**Pull request by Olya-web7**
[Edit child] The 'ЗБЕРЕГТИ' button stays enabled when mandatory 'По-батькові' field is empty #962

### CODE REVIEW

File: src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ export class CreateChildComponent extends CreateFormComponent implements OnInit,
     this.ChildrenFormArray.valueChanges.pipe(
       takeUntil(this.destroy$),
     ).subscribe((val: boolean) => {      
-      this.isEmpty = !Boolean(val[0].lastName) || !Boolean(val[0].firstName);
+      this.isEmpty = !!!(val[0].lastName) || !!!(val[0].firstName) || !!!(val[0].middleName);
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts:76]: !val[0].lastName
!val[0].firstName
!val[0].middleName

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts (hunk 1 above)
Comment (litvinets) [reply src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts:76]: wrong typo? Why val is boolean but has lastName, firstName?

Comment (litvinets) [reply src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts:76]: I think you should simplify is to !(val[0].lastName || val[0].firstName || val[0].middleName)

File: src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts [hunk 2]
```diff
@@ -73,7 +73,7 @@ export class CreateChildComponent extends CreateFormComponent implements OnInit,
     this.ChildrenFormArray.valueChanges.pipe(
       takeUntil(this.destroy$),
     ).subscribe((val: boolean) => {
```
Comment (litvinets) [inline src/app/shell/personal-cabinet/parent/create-child/create-child.component.ts:75]: change please typo

**Review by litvinets — CHANGES_REQUESTED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by dmitryy90 — APPROVED** [review]

**Review by litvinets — APPROVED** [review]

**Comment by litvinets** [comment]
link this issue please (https://github.com/ita-social-projects/OoS-Frontend/issues/965)

**Comment by litvinets** [comment]
https://github.com/ita-social-projects/OoS-Frontend/issues/965

**Pull request merged**